### PR TITLE
[stable/mongodb] replicaset.useHostnames option

### PR DESCRIPTION
**Pull request by dennybaa**
Added option to use bitnami `MONGODB_ADVERTISED_HOSTNAME` environment variable, it's enabled by default.

### CODE REVIEW

File: stable/mongodb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mongodb
-version: 4.0.6
+version: 4.0.7
```
Comment (juan131) [inline stable/mongodb/Chart.yaml]: Please bump the minor version. This is not simple 'patch'

**Review by juan131 — COMMENTED** [review]
This is great! Please check my small comment. @tompizmor could you please also take a look?

File: stable/mongodb/Chart.yaml (hunk 1 above)
Comment (dennybaa) [reply stable/mongodb/Chart.yaml]: np done

**Review by tompizmor — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**